### PR TITLE
ZW green text

### PR DESCRIPTION
**Pull request by zelunw**
fghcbrreafbgbesfsvafsdgergfd

### CODE REVIEW

File: src/DemoPackagePM.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module DemoPackagePM
 
-print_greeting() = print("Hello, world!")
+print_greeting() = printstyled("Hello, world!"; color= :green)
```
Comment (pcmaaaaa) [inline src/DemoPackagePM.jl]: I like iron man more than Hulk.

**Review by pcmaaaaa — COMMENTED** [review]
Perfect!

**Comment by timholy** [comment]
Bump, looks like it just needs to be merged. Do this before taking action on #5

**Comment by pcmaaaaa** [comment]
Thanks pal!

**Comment by timholy** [comment]
Looks like this was submitted against `PM/1stBranch` instead of `main`. Usually, pull requests are submitted against `main`, but this may have been a timing-of-merge limited thing.

**Pull request merged**